.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,110 +0,0 @@
-name: "\U0001F41B Bug Report"
```

Comment:
@maxin-cn It seems like you accidentally deleted the entire `.github` folder that we have. Could you please revert these changes?

Reply:
Hi @a-r-r-o-w . I am sorry for this. I have recovered them.